modules/op-specifying-pvc-as-volumesource-in-workspaces.adoc

```diff
@@ -0,0 +1,41 @@
+// This module is included in the following assembly:
+//
+//  *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
+
+[id="specifying-pvc-as-volumesource-in-workspaces_{context}"]
+= Specifying PersistentVolumeClaims as VolumeSource in Workspaces
+
+Workspaces help share data among Tasks and allow you to specify one or more volumes that each Task in the Pipeline requires during execution.
```

Comment:
s/Workspaces help share data among Tasks and /Workspaces help Tasks share data, and/